alf/algorithms/data_transformer_test.py

```diff
@@ -226,8 +226,7 @@ def test_compute_her_future_step_distance(self, end_prob):
             data_spec=self.data_spec,
             num_environments=num_envs,
             max_length=max_length,
-            keep_episodic_info=True,
-            step_type_field="t")
```

Comment:
Why did you have this `step_type_field` in the first place?

Reply:
So the test can have a different field name for step_type, which is no longer used due to this PR.
Just removed it.